### DISCUSSION TITLE
Login verification email not sent

### DISCUSSION THREAD

**Discussion by ihsaan-ullah** [community/community#118771] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am using a gmail account to login to github. After several tries, I am not receiving login verification emails. I tried forgot password but no email is received in my mailbox. I have checked spam+trash but nothing is there.

Note: I have previously received emails when I created the GitHub account.

**Comment by git-sumana** [comment]
1. Check your spam box
2. change your email address again in the GitHub Settings

**Reply by ihsaan-ullah** [reply]
Nothing in my spam.

How can I change email if I cannot login?

**Comment by git-sumana** [comment]
Don't you have passkeys saved earlier?

**Reply by ihsaan-ullah** [reply]
Nope, I haven't activated pass keys. Just email login is activated

**Comment by ihsaan-ullah** [comment]
I think it was a temporary glitch. Received emails after 4 hours of delay.
